website/docs/developer/api/api-gateway.md

```diff
@@ -49,10 +56,38 @@ The API invoke URL can be found under `Stages` section in API gateway UI. All AP
 }
 
 ```
+where
+- anotherPage(Boolean)- Indicates if more results can be retrieved on another page.
+- providerId(str)- The unique ID of the provider making the request.
+- collectionId(str)- The unique ID of collection to compare.
+- id(str)- The unique ID of the granule.
+- createdAt(int)- The time, in milliseconds since 1 January 1970 UTC, data was originally ingested into cumulus.
+- executionId(str)- Step function execution ID from AWS.
+- ingestDate(int)- The time, in milliseconds since 1 January 1970 UTC, that the data was originally ingested into ORCA.
+- lastUpdate(int)- The time, in milliseconds since 1 January 1970 UTC, that information was updated.
+- name(str)- The name and extension of the file.
+- cumulusArchiveLocation(str)- Cumulus bucket the file resides in.
+- orcaArchiveLocation(str)- ORCA S3 Glacier bucket the file resides in.
+- keyPath(str)- S3 path to the file including the file name and extension, but not the bucket.
+- sizeBytes(int)- Size in bytes of the file. From Cumulus ingest.
+- hash(str)- Checksum hash of the file provided by Cumulus.
+- hashType(str)- Hash type used to calculate the hash value of the file.
+- version(str)- AWS provided version of the file.
+
+
+## Recovery granules API
+
+The `recovery/granules` API call returns detailed status of the granule.
```

Comment:
Add that this relates to an ORCA recovery job status.

Reply:
added